test/DeltaTriplesTest.cpp

```diff
@@ -167,11 +167,11 @@ TEST_F(DeltaTriplesTest, insertTriplesAndDeleteTriples) {
   // Inserting unsorted triples works.
   deltaTriples.insertTriples(
       cancellationHandle,
-      makeIdTriples(vocab, localVocab, {"<B> <D> <C>", "<B> <C> <D>"}));
+      makeIdTriples(vocab, localVocab, {"<B> <C> <D>", "<B> <D> <C>"}));
```

Comment:
The comment above states `Inserting unsorted triples works` which is clearly out of date:) please adapt.